assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -27,25 +27,25 @@ command: >
   $[[--hf-tokenizer-class ${{inputs.hf_tokenizer_class}}]]
   $[[--hf-use-experimental-features ${{inputs.hf_use_experimental_features}}]]
   $[[--extra-pip-requirements "${{inputs.extra_pip_requirements}}"]]
-  --mlflow-flavor ${{inputs.mlflow_flavor}}
+  --model-framework ${{inputs.model_framework}}
   --model-path ${{inputs.model_path}}
   --mlflow-model-output-dir ${{outputs.mlflow_model_folder}}
   --model-import-job-path ${{outputs.model_import_job_path}}
 
 inputs:
   model_id:
     type: string
-    description: Huggingface model id (https://huggingface.co/<model_id>). A required parameter for transformers flavor. Can be provided as input here or in model_download_metadata JSON file.
+    description: Huggingface model id (https://huggingface.co/<model_id>). A required parameter for Huggingface model framework. Can be provided as input here or in model_download_metadata JSON file.
     optional: true
 
-  mlflow_flavor:
+  model_framework:
     type: string
     enum:
-      - transformers
-      - mmlab_pyfunc
-    default: transformers
+      - Huggingface
+      - mmlab
```

Comment:
Could we follow similar casing for both?

Reply:
Updated. Thanks for the suggestion.